idunn/blocks/wikipedia.py

```diff
@@ -108,6 +113,46 @@ def wrapper(*args, **kwargs):
 
         return wrapper
 
+class WikipediaCache:
+    _cache = None
```

Comment:
I think we should add a setting to set the expire timeout of values that are stored in Redis.
24h is probably a good default for wikipedia values.